Formula/apibuilder-cli.rb

```diff
@@ -0,0 +1,41 @@
+class ApibuilderCli < Formula
+  desc "Command-line interface to apibuilder"
+  homepage "https://www.apibuilder.io"
+  url "https://github.com/apicollective/apibuilder-cli/archive/0.1.4.tar.gz"
+  sha256 "8267dce31b951987ae2395be23180fe330ea2056c2f765ecb066736713378294"
+
+  def install
+    inreplace "bin/apibuilder", "File.join(File.dirname(__FILE__), 'apibuilder-cli-config')", "'#{prefix}/scripts/apibuilder-cli.rb'"
+    inreplace "bin/read-config", "File.join(File.dirname(__FILE__), 'apibuilder-cli-config')", "'#{prefix}/scripts/apibuilder-cli.rb'"
+
+    bin.install "bin/apibuilder"
+    bin.install "bin/read-config"
+    (prefix/"scripts").install Dir["lib/*"]
+  end
+
+  def post_install
+    ohai "Create user token at http://www.apibuilder.io/tokens"
+    ohai "Then create config file at ~/.apibuilder/config"
+    s = <<-EOS.undent
+     Example File:
+
+     [default]
+     token = <your API token>
+    EOS
+    ohai s
+
+    ohai "Full instructions at https://github.com/apicollective/apibuilder-cli"
+  end
+
+  test do
+    touch "config"
+    s = <<-EOS.undent
+    [default]
+    token = <your API token>
+   EOS
+    open("config", "w") do |f|
+      f.puts s
+    end
+    system "#{bin}/read-config", "--path", "config"
```

Comment:
The test needs to exercise and verify the functionality of the software once installed, not just check for 0 exit code.

Reply:
fixed with updated commit and https://github.com/apicollective/apibuilder-cli/pull/22